scripts/migrations.js

```diff
@@ -0,0 +1,79 @@
+import Constants from '../constants.js';
```

Comment:
The indentation of this file is screwed up. Should be 2 spaces. I can fix myself after merge if it's a problem